src/legacy/ui/public/vis/components/tooltip/_hierarchical_tooltip.html

```diff
@@ -16,8 +16,12 @@
   </thead>
   <tbody>
     <tr class="visTooltip__value" ng-repeat="row in rows track by $index">
-      <td><span ng-bind-html="row.spacer"></span>{{row.field}}</td>
-      <td>{{row.bucket}}</td>
+      <td>
```

Comment:
Just curious, is there a reason for separating these into two `<td>` elements? Seem like the same thing with just a single `<td>` element.

Reply:
I'm not sure if I follow you, but actually those are two different columns: one is the field selected (aka the bucket field), one is the value of that bucket and there is the third column that is the metric associated with that bucket.
If you have multiple levels on the same piechart, than you can distinguish the levels looking at the first column:
<img width="355" alt="Screenshot 2019-07-31 at 09 57 52" src="https://user-images.githubusercontent.com/1421091/62194243-c63fde80-b379-11e9-8d6b-e1caa5032487.png">

Reply:
I see, nevermind I was thinking of something else. 👍